Dockerfile

```diff
@@ -1,82 +1,123 @@
-FROM ruby:3.3-slim-bullseye
-
+FROM ruby:3.3-slim-trixie AS base
 LABEL maintainer="Stuart Owen <orcid.org/0000-0003-2130-0865>, Finn Bacall"
 ARG SOURCE_COMMIT
-ENV APP_DIR=/seek
-ENV RAILS_ENV=production
+
+ENV APP_DIR=/seek \
+    RAILS_ENV=production \
+    BUNDLE_PATH="/usr/local/bundle" \
+    BUNDLE_WITHOUT="development"
+
+# Supercronic variables
+ENV SUPERCRONIC_URL=https://github.com/aptible/supercronic/releases/download/v0.1.12/supercronic-linux-amd64 \
+    SUPERCRONIC=supercronic-linux-amd64 \
+    SUPERCRONIC_SHA1SUM=048b95b48b708983effb2e5c935a1ef8483d9e3e
+
 
 # need to set the locale, otherwise some gems file to install
 ENV LANG="en_US.UTF-8" LANGUAGE="en_US:UTF-8" LC_ALL="C.UTF-8"
 
+# Install base dependencies
 RUN apt-get update -qq && \
-    apt-get install -y --no-install-recommends build-essential cmake curl default-mysql-client gettext graphviz git \
-		libcurl4-gnutls-dev libmagick++-dev libmariadb-dev libpq-dev libreadline-dev \
-		libreoffice libsqlite3-dev libssl-dev libxml++2.6-dev \
-		libxslt1-dev libyaml-dev locales nginx nodejs openjdk-11-jdk-headless \
-		python3.9-dev python3.9-distutils python3-pip \
-		poppler-utils postgresql-client shared-mime-info sqlite3 links telnet vim-tiny zip && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/* && \
-    locale-gen en_US.UTF-8
+    apt-get install --no-install-recommends -y \
+    curl zip libjemalloc2 libvips \
+    sqlite3 postgresql-client default-mysql-client \
+    vim-tiny git openjdk-21-jre python3.13 \
+    poppler-utils graphviz gettext shared-mime-info
 
+# Prepare app directory
 RUN mkdir -p $APP_DIR
-RUN chown -R www-data $APP_DIR /var/www
+RUN chown www-data $APP_DIR
+WORKDIR $APP_DIR
+
+# Disable ssl from the mysql client
+RUN echo "[client]\nskip-ssl" > /etc/mysql/conf.d/disable-ssl.cnf
+
+FROM base AS builder
+
+# Install build dependencies
+RUN apt-get update -qq && \
+    apt-get install -y --no-install-recommends build-essential cmake \
```

Comment:
also keep alphabetical. Could these all be moved to the base? Would easier to maintain if all in one place